server/src/main/java/org/jboss/as/server/deployment/Phase.java

```diff
@@ -380,6 +380,7 @@ public Phase next() {
     public static final int DEPENDENCIES_SEAM                           = 0x0A01;
     public static final int DEPENDENCIES_WS                             = 0x0C00;
     public static final int DEPENDENCIES_SECURITY                       = 0x0C50;
+    public static final int DEPENDENCIES_ELYTRON                        = 0x0C51;
```

Comment:
To follow up on this for the record, since the new one is a replacement for the one before it in the sequence, there is no use case for inserting others in between so a gap is not needed.

And if some day that proves wrong it's always possible to change the sequence number.